File: app/src/test/java/com/makingiants/todayhistory/api/MockHistory.java
package com.makingiants.todayhistory.api;

import com.makingiants.today.api.repository.history.pojo.Event;
import java.util.ArrayList;
import java.util.List;

public class MockHistory {
  public static List<Event> events(int number) {
    List<Event> list = new ArrayList<>(number);
    for (int i = 0; i < number; i++) {
      String title = String.format("Title %d", i);
      String date = String.format("1/1/%d", i);
      String imageUrl = String.format("http://image.com/%d", i);

      list.add(new Event(title, date, imageUrl));
    }
    return list;
  }
}

File: app/src/test/java/com/makingiants/todayhistory/api/MockHistory.kt
package com.makingiants.todayhistory.api

import com.makingiants.today.api.repository.history.pojo.Event
import com.makingiants.today.api.repository.history.pojo.Thumb
import java.util.*

object MockHistory {
  fun events(number: Int): List<Event> {
    val list = ArrayList<Event>(number)
    for (i in 0..number - 1) {
      val title = String.format("Title %d", i)
      val date = String.format("1/1/%d", i)
      val image = Thumb(i, i, "http://image.com/%d")

      list.add(Event(title, date, image))
    }
    return list
  }
}
